flixel/system/FlxAssets.hx

```diff
@@ -28,7 +28,7 @@ class GraphicLogo extends BitmapData {}
 class GraphicVirtualInput extends BitmapData {}
 
 @:file("assets/images/ui/virtual-input.txt")
-class VirtualInputData extends ByteArray {}
+class VirtualInputData extends #if lime_legacy ByteArray #else ByteArrayData #end {}
```

Comment:
See this PR to OpenFL that fixes a few things. Also if I bump the internal version number I can do a proper check to maintain compatibility.

https://github.com/openfl/openfl/pull/952